src/SFA.DAS.EmployerAccounts/Models/Role.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerAccounts.Models
+{
+    public enum Role
+    {
+        None = 0,
+        Owner = 1,
+        Transactor = 2,
+        Viewer = 3
+    }
+
+    public static class RoleStrings
+    {
+        public static string GetRoleDescription(Role role)
+        {
+            return GetRoleDescription(role.ToString());
+        }
+
+        public static string GetRoleDescription(string role)
+        {
+            switch (role)
+            {
+                case "Owner": return "Accept agreements, view information and manage PAYE schemes, organisations, apprentices and team members";
+                case "Transactor": return "Add apprentices and view information";
+                case "Viewer": return "View information but can’t make changes";
+                default: throw new ArgumentException("Unexpected role: " + role);
+            }
+        }
+
+        public static string GetRoleDescriptionToLower(Role role)
+        {
+            return GetRoleDescriptionToLower(role.ToString());
+        }
+
+        public static string GetRoleDescriptionToLower(string role)
+        {
+            var str = GetRoleDescription(role);
+            return char.ToLower(str[0]) + str.Substring(1);
+        }
+    }
+}
```

Comment:
my own lack of understanding here, what is the purpose of this file being added?

Reply:
It, bizarrely, existed in the old implementation of SFA.DAS.Authorization so I've had to move it out.  It is also copied into SFA.DAS.EAS.Domain but that will be tidied up when we remove all of the remaining redundant code.